pointercrate-demonlist-pages/src/account/list_integration.rs

```diff
@@ -177,6 +177,10 @@ impl AccountPageTab for ListIntegrationTab {
                         br;
                         "To initiate a claim, click the pen left of the 'Claimed Player' heading. Once initiated, you have an unverified claim on a player. These claims will then be manually verified by members of the pointercrate team. You can request verification in " a.link href=(self.0) {"this discord server"} "."
                         br;
+                        b {
+                            "Note: for all list related inquiries including but not limited to, record status, name changes, etc, you should use " a.link href=(self.1) {"this discord server"} "."
```

Comment:
The demonlist discord is also hardcoded over at https://github.com/stadust/pointercrate/blob/bfa633b57decf3063b69a066e9491cebd2ab82c9/pointercrate-demonlist-pages/src/lib.rs#L188-L197, I think for now its fine to do the same here, if only to reduce the churn of coordinating an update with the private repository